packages/animation-curve-block/src/components/AnimationCurveFlyout.tsx

```diff
@@ -0,0 +1,235 @@
+    const [x2Validation, setX2Validation] = useState<Validation>(Validation.Default);
+    const [localAnimationFunction, setLocalAnimationFunction] = useState(animationCurve.animationFunction);
+
+    useEffect(() => {
+        if (localAnimationFunction.parameters.x1) {
+            if (localAnimationFunction.parameters.x1 < 0 || localAnimationFunction.parameters.x1 > 1) {
+                setX1Validation(Validation.Error);
+            } else {
+                setX1Validation(Validation.Default);
+            }
+        }
+        if (localAnimationFunction.parameters.x2) {
+            if (localAnimationFunction.parameters.x2 < 0 || localAnimationFunction.parameters.x2 > 1) {
+                setX2Validation(Validation.Error);
+            } else {
+                setX2Validation(Validation.Default);
+            }
+        }
+    }, [localAnimationFunction]);
+
+    const handleSave = () => {
+        onSave(animationCurve.id, { ...animationCurve });
+    };
+
+    const updateLocalCurve = (animationCurvePatch: AnimationCurve): void => {
+        animationCurvePatch.animationFunction.parameters.x1 =
+            Math.round(animationCurvePatch.animationFunction.parameters.x1 * 100) / 100;
+        animationCurvePatch.animationFunction.parameters.x2 =
+            Math.round(animationCurvePatch.animationFunction.parameters.x2 * 100) / 100;
+        animationCurvePatch.animationFunction.parameters.y1 =
+            Math.round(animationCurvePatch.animationFunction.parameters.y1 * 100) / 100;
+        animationCurvePatch.animationFunction.parameters.y2 =
+            Math.round(animationCurvePatch.animationFunction.parameters.y2 * 100) / 100;
+        setLocalAnimationFunction(animationCurvePatch.animationFunction);
+        if (![x1Validation, x2Validation].includes(Validation.Error)) {
+            onAnimationCurveUpdate &&
+                onAnimationCurveUpdate(animationCurve.id, {
+                    ...animationCurve,
+                    ...animationCurvePatch,
+                });
+
+            onAnimationCurveChange && onAnimationCurveChange({ ...animationCurve, ...animationCurvePatch });
+        }
+    };
+
+    const updateAnimationFunction = (animationFunctionPatch: AnimationFunctionPatch): void => {
+        updateLocalCurve({
+            ...animationCurve,
+            animationFunction: {
+                ...animationCurve.animationFunction,
+                ...animationFunctionPatch,
+            },
+        });
+    };
+
+    const updateAnimationCurveParameters = (parametersPatch: AnimationCurveParametersPatch): void => {
+        updateAnimationFunction({
+            parameters: {
+                ...animationCurve.animationFunction.parameters,
+                ...parametersPatch,
+            },
+        });
+    };
+
+    return (
+        <Flyout
+            trigger={<></>}
+            isOpen={isFlyoutOpen}
+            onOpenChange={onOpenChange}
+            onCancel={onCancel}
+            legacyFooter={false}
```

Comment:
yeah that prop is deprecated but if I remove it, it will use the legacyFooter.

Because by default there is legacyFooter: true (otherwise that prop wouldn't not be needed) https://github.com/Frontify/fondue/blob/c8978e54e938a9ec74df0b3f128219213ef3b1f5/src/components/Flyout/Flyout.tsx#LL80C10-L80C10

I assume they "solved" it that way because of compatibility issues, but its just guessing from my side